app/helpers/file_helper.rb

```diff
@@ -2,9 +2,12 @@ module FileHelper
   def syntax_highlighter_for_filename(filename, track)
     parts = filename.split(".")
 
-    lang = parts.size > 1 ?
-      Exercism::PrismFileMappings[parts.last.downcase] :
-      track.syntax_highligher_language
+    if !parts.empty?
+      extension = parts.last.downcase
+      lang = Exercism::PrismFileMappings[extension]
+    end
+
+    lang ||= track.syntax_highligher_language
```

Comment:
I managed to fail to submit (and lose) the comment I was refereing to in the review. I suggest. I suggested that for this bit:

```ruby
if !parts.empty?
  extension = parts.last.downcase
  lang = Exercism::PrismFileMappings[extension]
end

lang ||= track.syntax_highligher_language
```

You could change it to:
```ruby
extension = parts.last.try(&:downcase)
lang = Exercism::PrismFileMappings[extension] ||
       track.syntax_highligher_language
```

I personally find that a bit more readable. But I'll leave you to choose :)

Reply:
So do I!